services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -493,6 +495,30 @@ public static Builder builder() {
   @Nullable
   public abstract WalkingOptions walkingOptions();
 
+  /**
+   * The desired arrival time, the local time at the route <b>destination</b>.
```

Comment:
```suggestion
   * The departure time, the local time at the route <b>destination</b>.
```

looks like docs for `departAt` and `arriveBy` are mixed in both `RouteOptions` and `RouteOptions.Builder`
and inside `MapboxDirections`

Reply:
yep, nice note. They're mixed.